### DISCUSSION TITLE
Is it possible ? Modify a file containing a macro by deleting this macro

### DISCUSSION THREAD

**Discussion by jcacqua** [decalage2/oletools#757] [Q&A]
Everything is in the title :-) : We would like to ask if it is, by any way, possible now to modify a file containing a macro , by only deleting the macro of this file ?

This is a function we would use as an automated/scripted action for these files.

Thanks for your help

**Comment by decalage2** [comment]
It is technically possible to implement it, but it's not done yet and it would require some work. Different methods are needed to handle the different file formats.
